QUEST_JOBSTEP.tsv

```diff
@@ -753,42 +753,42 @@ QUEST_JOBSTEP_20150317_000752	The disciples came to Lydia Schaffen. Afterwards,
+QUEST_JOBSTEP_20150317_000770	$After Rumpelstiltskin seemingly honest answer, Lydia Schaffen replied, 'Alright. One more question though. In a place like this, what sort of targets will I shoot at? I mean you won't be used to riding the arrows until round 2 or round 3, so the targets for each round should be called in advance. Don't you agree?'{nl}'Alright, let's do that,' agreed Rumpelstiltskin. {nl}And after agreeing on compensation upon defeat as well, the game between two people, err, one person and one demon began.{nl}
+QUEST_JOBSTEP_20150317_000771	$Lydia Schaffen weighed her arrow, then called out the first target,{nl}'The tawny tree over there!'{nl}Together with those words in advance she fired her bowstring, the arrow's frightful speed sliced through the air.{nl}
+QUEST_JOBSTEP_20150317_000772	$Rumpelstiltskin moved nimbly to chase the flying arrow, and after he got on, the arrow impaled the tree. As the arrow continued to oscillate, he danced on the arrow and sang an unknown tune.{nl}Lydia Schaffen again gauged her arrows, then she called,{nl}'All of the pinecones from the pine tree on my front left side!'{nl}Upon calling the target, Lydia Schaffen unveiled her godlike archery skills. 
+QUEST_JOBSTEP_20150317_000773	$Her arrows poured faster than heavy rain during a hot summer day, as if the white tailed arrows were fired at the same time in the same direction. {nl}
+QUEST_JOBSTEP_20150317_000774	$Rumpelstiltskin couldn't take it as calmly this round. Faster than the human eye can distinguish, Rumpelstiltskin sat on each arrow one by one and transfer to another arrow as the barrage cuts through the air. In the end, when he reached the last arrow before it reached the last pinecone, he was concerned he might crash into the target.{nl}Rumpelstiltskin, in his tired state, worked up a triumphant expression and spoke to Lydia Schaffen.{nl}
+QUEST_JOBSTEP_20150317_000775	$'So anyways, I managed to sit on every single arrow before the last arrow reached the target, therefore I win.'{nl} Lydia Schaffen replied,{nl}' Certainly, I can't deny your word that you sat on every arrow before the last one reached it target.'{nl}
+QUEST_JOBSTEP_20150317_000776	$Rumpelstiltskin gave a wicked grin after hearing Lydia Schaffen's words and said,{nl}'Therefore, I win this game. Well then, let's talk about the agreed compensation…'{nl}Before Rumpelstiltskin could finish, Lydia Schaffen shook her head and spoke,{nl}'Don't tell me that because you won 2 rounds, you won't play the final round? I'm fine not doing it, but clearly we did promise each other to play 3 rounds, right?'{nl}
+$QUEST_JOBSTEP_20150317_000777	$'The game doesn't end before the 3rd round, so I think it's pointless to talk about 'compensation' without the game ending…'{nl}
```

Comment:
Typo: you have a $ at the start of the line AND at the start of the translation. Remove the one at the start of the line, please.